### PR TITLE
Feature/alcs 545

### PR DESCRIPTION
**Pull request by mhuseinov**
* toggle buttons validation error
* checkbox on view/edit/submit pages
* file upload validation errors
* fix issue with disabled owner validation

### CODE REVIEW

File: portal-frontend/src/app/features/application-details/parcel/parcel.component.ts [hunk 1]
```diff
@@ -18,14 +18,15 @@ export class ApplicationParcelBasicValidation {
...
 
-  isTypeRequired = false;
-  isPidRequired = false;
-  isPinRequired = false;
-  isLegalDescriptionRequired = false;
-  isMapAreaHectaresRequired = false;
-  isPurchasedDateRequired = false;
-  isFarmRequired = false;
-  isCertificateRequired = false;
+  isTypeRequired: boolean = false;
```
[the remark this answers is not in the document]
Comment (mhuseinov) [reply portal-frontend/src/app/features/application-details/parcel/parcel.component.ts:21]: This is what happens when 2 people edit the same code 🙂 I'll double check it

File: portal-frontend/src/app/features/edit-application/other-parcels/other-parcels.component.html [hunk 2]
```diff
@@ -84,6 +88,55 @@ <h2>Other Parcels in the Community</h2>
         </button>
       </div>
     </div>
+    <div class="full-row">
```
[the remark this answers is not in the document]
Comment (mhuseinov) [reply portal-frontend/src/app/features/edit-application/other-parcels/other-parcels.component.html:91]: hmm..., it looks like exact copy, probably another issue of merge

File: portal-frontend/src/app/features/application-details/parcel/parcel.component.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (mhuseinov) [reply portal-frontend/src/app/features/application-details/parcel/parcel.component.ts:21]: just checked, I can see exactly the same code on dev except of the isConfirmedByApplicant that I've added in scope of my ticket

**Review by dhaselhan — APPROVED** [review]
LGTM 👍

**Pull request merged**